paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/playground/activity/FawryActivity.kt

```diff
@@ -50,6 +50,12 @@ class FawryActivity : AppCompatActivity() {
         }
     }
 
+    override fun onDestroy() {
+        super.onDestroy()
+
+        ExternalPaymentMethodResultHandler.onExternalPaymentMethodResult(this, ExternalPaymentMethodResult.canceled())
```

Comment:
Should we assume it's cancelled if our external proxy activity sees onResume after launching the merchants EPM activity?

Reply:
I think that probably doesn't work if the merchant doesn't launch an activity in their external payment method implementation. Because then onResume() could also be called if a user were to exit the app and come back to it, right?

I thought about doing this and have a PR that does it, but I think it's fairly fragile/likely to not work well for all integrations. Definitely happy to discuss it though, this feels like an interesting problem

Reply:
Makes sense. I agree. Thanks for considering!